src/plugins/controls/public/controls/timeslider_control/get_timeslider_control_factory.tsx

```diff
@@ -270,7 +271,10 @@ export const getTimesliderControlFactory = (): ControlFactory<
         Component: (controlPanelClassNames) => {
           const [isAnchored, isPopoverOpen, timeRangeMeta, timeslice] =
             useBatchedPublishingSubjects(isAnchored$, isPopoverOpen$, timeRangeMeta$, timeslice$);
-
+          const [localTimeslice, setLocalTimeslice] = useState<Timeslice>([
+            timeRangeMeta.timeRangeMin,
+            timeRangeMeta.timeRangeMax,
+          ]);
```

Comment:
Since this is only relevant and useful to the `TimeSliderPopoverContent` component, is there a reason we are't defining it there instead? Why define it for the whole control?